File: .cache/download_sounds.js
const https = require('https');
const fs = require('fs');
const { execSync } = require('child_process');

const url = 'https://kenney.nl/assets/casino-audio';

https.get(url, (res) => {
  let data = '';
  res.on('data', chunk => { data += chunk; });
  res.on('end', () => {
    const match = data.match(/href="([^"]*\.zip)"/);
    if (match) {
      let zipUrl = match[1];
      if (!zipUrl.startsWith('http')) zipUrl = 'https://kenney.nl' + zipUrl;
      console.log('Found ZIP:', zipUrl);
      
      const file = fs.createWriteStream('casino.zip');
      https.get(zipUrl, (response) => {
        response.pipe(file);
        file.on('finish', () => {
          file.close();
          console.log('Downloaded ZIP. Extracting...');
          try {
            // PowerShell의 Expand-Archive 사용
            execSync('powershell -Command "Expand-Archive -Path casino.zip -DestinationPath casino_sounds -Force"', { stdio: 'inherit' });
            console.log('Extracted successfully.');
          } catch (e) {
            console.error('Extraction failed', e);
          }
        });
      }).on('error', (err) => {
        fs.unlink('casino.zip');
        console.error('Error downloading zip:', err.message);
      });
    } else {
      console.log('ZIP not found in HTML.');
    }
  });
}).on('error', (err) => {
  console.error('Error fetching page:', err.message);
});

File: .cache/download_sounds.py
import urllib.request
import zipfile
import os
import re

url = "https://kenney.nl/assets/casino-audio"
try:
    req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
    html = urllib.request.urlopen(req).read().decode('utf-8')
    match = re.search(r'href="([^"]*\.zip)"', html)
    if match:
        zip_url = match.group(1)
        if not zip_url.startswith('http'):
            zip_url = "https://kenney.nl" + zip_url
        print("Found ZIP:", zip_url)
        urllib.request.urlretrieve(zip_url, "casino.zip")
        with zipfile.ZipFile("casino.zip", 'r') as zip_ref:
            zip_ref.extractall("casino_sounds")
        print("Downloaded and extracted.")
    else:
        print("ZIP not found in HTML.")
except Exception as e:
    print("Error:", e)
